### ISSUE TITLE
Backup include downloaded songs

### ISSUE THREAD

**Issue by Anonuser-code** [z-huang/music#1436]
### Checklist

- [x] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

It would be way better if 'back up' option include downloaded songs. So we can restore all our songs.

### Why do you want this feature?

When i upgrade app, the downloaded songs remain but when i want to install previous version, i have to download my all songs. I would add my songs a YT connected playlist but it's a bit of a hassle to add every song that i download to a youtube playlist. So please add a feature to back up all songs, if possible. Thanks.

**Comment by z-huang** [comment]
Why would you downgrade the app?

**Comment by filinux13** [comment]
Yeah, sometimes when the songs got taken from YouTube for some reason it could be usefull, but i don't think a 4gb backup file could even load (songs can use a lot of space, you maybe want to download the songs and store then externaly?)

**Comment by Anonuser-code** [comment]
> Why would you downgrade the app?

It's not just about downgrade. The same thing happens when I delete the app for some reason. I should back up downloaded songs even i reinstall the app.
